client/src/pages/ProfileDetails/ProfileDetails.tsx

```diff
@@ -0,0 +1,101 @@
+import Grid from '@material-ui/core/Grid';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Avatar from '@material-ui/core/Avatar';
+import IconButton from '@material-ui/core/IconButton';
+import Rating from '@material-ui/lab/Rating';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import LocationOnIcon from '@material-ui/icons/LocationOn';
+import useStyles from './useStyles';
+import ProfileImageList from './ProfileImageList';
+import { useAuth } from '../../context/useAuthContext';
+import { User } from '../../interface/User';
+
+interface Props {
+  user: User;
+}
+
+export default function ProfileDetails(): JSX.Element {
+  const classes = useStyles();
+
+  const { loggedInUser } = useAuth();
+
+  return (
+    <Grid container component="main" className={classes.root}>
+      <CssBaseline />
+      <Grid container className={classes.cards}>
+        <Grid item className={classes.profileCard}>
+          <Card>
+            <CardActionArea>
+              <CardMedia
+                className={classes.media}
+                image="https://images.unsplash.com/photo-1570129477492-45c003edd2be?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=750&q=80"
+              />
+              <CardContent className={classes.cardContent}>
+                <Avatar className={classes.avatar} />
+                <Typography className={classes.nameField}>First Last</Typography>
+                <Typography className={classes.subtextField}>Loving pet sitter</Typography>
+                <IconButton className={classes.locationField} disabled>
+                  <LocationOnIcon color="primary" fontSize="small" />
+                  Location
+                </IconButton>
+                <Typography className={classes.aboutMeField}>About me</Typography>
+                <Typography>
+                  Lorem ipsum delorum Lorem ipsum delorum Lorem ipsum delorum Lorem ipsum delorum Lorem ipsum delorum
+                  Lorem ipsum delorum Lorem ipsum delorum Lorem ipsum delorum Lorem ipsum delorum Lorem ipsum
+                </Typography>
```

Comment:
It's okay to fill in fields with temporary values, however, hardcoding values manually like this can create a lot of overhead for integration. Instead, I recommend creating a mock-data that is identical/resembling of your actual model and fill in these details from there instead. E.g. something like:

```
const mockUser = {
    id: 0,
    firstName: "John",
    lastName: "Doe",
    about: "About me",
    bio: "Lorem Ipsum ...",
    ...,
  };
```
and later
```
<Typography className={classes.aboutMeField}>{mockUser.about}</Typography>
```